src/test/scala/org/clulab/reach/TestFeedback2.scala

```diff
@@ -56,8 +56,13 @@ class TestFeedback2 extends FlatSpec with Matchers {
   }
 
   val s7 = "We observed that endogenous ASPP2 translocates from cell/cell junctions to the cytosol/nucleus following RAS activation"
-  s7 should "contain 1 or more translocation events" in {
-    // TODO: we should pick 4 translocation events here - ENRIQUE
+  s7 should "contain 2 translocation events" in {
+    // TODO: we should pick 2 translocation events here - ENRIQUE
+    // ENRIQUE: Cell is in the black list of the NER, so there will be only two events
```

Comment:
It works correctly. There are two sources and two destinations, but one of the sources is `cell` which is filtered by the NER. So only two translocations match, where the source is `cell junctions` and the destinations are `cytosol` and `nucleus`